user_guide_src/source/database/queries.rst

```diff
@@ -240,6 +242,8 @@ close out the prepared statement when you're done with it:
 
 .. literalinclude:: queries/020.php
 
+.. note:: Since v4.3.0, the close() method deallocates the prepared statement in all DBMS. Previously, they were not deallocated in Postgre, SQLSRV and OCI8.
```

Comment:
```suggestion
.. note:: Since v4.3.0, the ``close()`` method deallocates the prepared statement in all DBMS. Previously, they were not deallocated in Postgre, SQLSRV and OCI8.
```

Reply:
done...